osu.Game.Rulesets.Osu/Difficulty/Skills/OsuSkill.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using osu.Game.Rulesets.Difficulty.Preprocessing;
+using osu.Game.Rulesets.Difficulty.Skills;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Osu.Difficulty.Preprocessing;
+using osu.Game.Rulesets.Osu.Objects;
+
+namespace osu.Game.Rulesets.Osu.Difficulty.Skills
+{
+    public abstract class OsuSkill : Skill
+    {
+        private readonly List<double> strains = new List<double>();
+        private readonly List<double> times = new List<double>();
+        private double target_fc_precision = 0.01;
```

Comment:
What is a reason to use snake_case, when you are using camelCase in another place for not const vars?